src-opam/distro.ml

```diff
@@ -1155,12 +772,18 @@ let distro_of_tag x : t option =
   | "opensuse-15.5" -> Some (`OpenSUSE `V15_5)
   | "opensuse-tumbleweed" -> Some (`OpenSUSE `Tumbleweed)
   | "opensuse" -> Some (`OpenSUSE `Latest)
-  | s when String.is_prefix ~affix:"cygwin-" s ->
-      win10_of_tag "cygwin-" s (fun v -> `Cygwin v)
-  | s when String.is_prefix ~affix:"windows-mingw-" s ->
-      win10_of_tag "windows-mingw-" s (fun v -> `Windows (`Mingw, v))
-  | s when String.is_prefix ~affix:"windows-msvc-" s ->
-      win10_of_tag "windows-msvc-" s (fun v -> `Windows (`Msvc, v))
+  | "cygwin-ltsc2016" -> Some (`Cygwin `Ltsc2016)
+  | "cygwin-ltsc2019" -> Some (`Cygwin `Ltsc2019)
+  | "cygwin-ltsc2022" -> Some (`Cygwin `Ltsc2022)
```

Comment:
This doesn't round-trip with the values above atm!

Reply:
Fixed in the latest commit.